simple_ado/utilities.py

```diff
@@ -18,7 +17,7 @@ def boolstr(value: bool) -> str:
 
 
 def download_from_response_stream(
-    *, response: requests.Response, output_path: str, log: logging.Logger
+        *, response: requests.Response, output_path: str, log: logging.Logger, callback=None 
```

Comment:
Can you fix the alignment and add the types to the callback please? (Running `./stylecheck.sh` should tell you about these things).